dat/mission.xml

```diff
@@ -1041,6 +1041,29 @@
    <planet>Gastan</planet>
   </avail>
  </mission>
+ <mission name="Novice Nebula Research">
+  <lua>zalek/neburesearch/neburesearch_00</lua>
+  <flags>
+   <unique />
+  </flags>
+  <avail>
+   <chance>100</chance>
+   <location>Bar</location>
+   <planet>Jorla</planet>
+  </avail>
+ </mission>
+ <mission name="Advanced Nebula Research">
+  <lua>zalek/neburesearch/neburesearch_01</lua>
+  <flags>
+   <unique />
+  </flags>
+  <avail>
+   <cond>player.misnDone("Novice Nebula Research")</cond>
```

Comment:
There is a better way to do that with the `<done>` flag. I'll recommend using that for readability.